ee/modules/030-cloud-provider-vsphere/docs/CONFIGURATION.md

```diff
@@ -17,18 +17,17 @@ If you need to configure a module because, say, you have a bare metal cluster an
 * `host` — the domain of the vCenter server;
 * `username` — the login ID;
 * `password` — the password;
-* `vmFolderPath` — the path to the VirtualMachine Folder where the cloned VMs will be created;
-  * e.g., `dev/test`;
 * `insecure` — can be set to `true` if vCenter has a self-signed certificate;
   * Format — bool;
   * An optional parameter; by default `false`;
+* `vmFolderPath` — the path to the VirtualMachine Folder where the cloned VMs will be created;
```

Comment:
After adding the openapi schema for the module, you can autogenerate the module reference. See the example below https://github.com/deckhouse/deckhouse/blob/3e7ce5b1433be2ac856bee5db3638fdc2c5c5c46/ee/fe/modules/340-extended-monitoring/docs/CONFIGURATION.md?plain=1#L5-L8